merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/LocalMissionModelService.java

```diff
@@ -213,9 +229,8 @@ public void refreshActivityTypes(final String missionModelId)
   throws NoSuchMissionModelException
   {
     try {
-      final var activityTypes =
-          loadUnconfiguredMissionModel(missionModelId)
-              .getActivityTypes();
+      final var activityTypes = this.loadUnconfiguredMissionModel(missionModelId)
+                                    .getActivityTypes();
```

Comment:
This is also just a whitespace change, yes?

Reply:
Yes - see commit called "Minor style tweaks to LocalMissionModelService" - I'd also be okay with dropping that commit if you don't think it adds much value.

Reply:
Nah, I'm fine with whitespace changes. I just wanted to make sure my eyes weren't playing tricks on me. Thanks for factoring them out!